app/src/renderer/vuex/modules/wallet.js

```diff
@@ -12,13 +12,11 @@ export default ({ node }) => {
     denoms: [],
     accountNumber: null,
     address: null,
-    subscribedRPC: null
+    subscribedRPC: null,
+    externals: { config }
```

Comment:
should we copy the config in each module or have a central point to access it? Could be a getter i.e..

Reply:
yeah that was my experiment with externals, in the end it seemed more complex because config and axios basically have different needs
I should have it stashed (almost working). I can push it on a separate PR, it changes many other places too.